Resources/public/javascript/sticky.js

```diff
@@ -0,0 +1,34 @@
+require('@wearejust/sticky');
+
+let addBtnClone;
+let elOffsetLeft;
+let $window = $(window);
+let minWidth = 940;
+
+$(function() {
+	if($window.width() > minWidth) {
```

Comment:
Als je bij initialisatie al checkt op grootte, werkt het niet als je op desktop een klein scherm groot trekt. De width vergelijking **altijd** in een resize functie stoppen.